libs/pika/command_line_handling/src/late_command_line_handling.cpp

```diff
@@ -33,7 +33,7 @@ namespace pika::detail {
         return str;
     }
 
-    int handle_late_commandline_options(util::runtime_configuration& ini,
+    int handle_late_commandline_options(pika::util::runtime_configuration& ini,
```

Comment:
Just curious, did you add this here and elsewhere because things weren't compiling without the extra `pika::` or just to be explicit (I'm asking since quite a few of these changes touch files that have nothing to do with the runtime module)? Not asking for a change here now.